### PR TITLE
♻️ Enable `import/order` on not-recently-updated extensions

### PR DESCRIPTION
**Pull request by rcebulko**
Turns on the `import/order` eslint rule. See the first commit for the "real" config changes; the second commit is the result of `amp lint --fix`.

This change excludes all extension directories that have been modified in the last 2 weeks to minimize disruption on "active" extension development.

```javascript
// These extensions have been modified in the last 2 weeks, so we're not
// including them yet to minimize disruption to devs actively working on them.
const EXCLUDED_EXTENSIONS = [
  'amp-ad-network-valueimpression-impl',
  'amp-ad',
  'amp-addthis',
  'amp-analytics',
  'amp-anim',
  'amp-apester-media',
  'amp-autocomplete',
  'amp-base-carousel',
  'amp-brightcove',
  'amp-carousel',
  'amp-connatix-player',
  'amp-consent',
  'amp-dailymotion',
  'amp-facebook-comments',
  'amp-facebook-like',
  'amp-facebook-page',
  'amp-facebook',
  'amp-fit-text',
  'amp-iframe',
  'amp-inline-gallery',
  'amp-lightbox',
  'amp-link',
  'amp-minute-media-player',
  'amp-mustache',
  'amp-nexxtv-player',
  'amp-recaptcha-input',
  'amp-render',
  'amp-sidebar',
  'amp-story-auto-ads',
  'amp-story',
  'amp-story',
  'amp-stream-gallery',
  'amp-subscriptions',
  'amp-tiktok',
  'amp-twitter',
  'amp-video',
];
```

### CODE REVIEW

File: extensions/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  'amp-dailymotion',
+  'amp-facebook-comments',
+  'amp-facebook-like',
+  'amp-facebook-page',
+  'amp-facebook',
+  'amp-fit-text',
+  'amp-iframe',
+  'amp-inline-gallery',
+  'amp-lightbox',
+  'amp-link',
```
Comment (alanorozco) [inline extensions/.eslintrc.js:41]: `amp-link` does not exist, did you mean?
```suggestion
  'amp-link-rewriter',
```

File: extensions/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  'amp-minute-media-player',
+  'amp-mustache',
+  'amp-nexxtv-player',
```
Comment (alanorozco) [inline extensions/.eslintrc.js:44]: The change to this extension was minor and unrelated to JS. Consider removing.
```suggestion
```

**Review by alanorozco — APPROVED** [review]

File: extensions/.eslintrc.js (hunk 2 above)
Comment (rcebulko) [reply extensions/.eslintrc.js:44]: Done.

File: extensions/.eslintrc.js (hunk 1 above)
Comment (rcebulko) [reply extensions/.eslintrc.js:41]: fixed

File: extensions/.eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  'amp-recaptcha-input',
+  'amp-render',
+  'amp-sidebar',
+  'amp-story-auto-ads',
+  'amp-story',
+  'amp-story',
```
[the remark this answers is not in the document]
Comment (rcebulko) [reply extensions/.eslintrc.js:50]: fixed, dropped the `-interactive`

File: extensions/.eslintrc.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// These extensions have been modified in the last 2 weeks, so we're not
+// including them yet to minimize disruption to devs actively working on them.
+const EXCLUDED_EXTENSIONS = [
```
[the remark this answers is not in the document]
Comment (rcebulko) [reply extensions/.eslintrc.js]: Done

**Pull request merged**